scripts/test-runner.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+sudo -E env LD_LIBRARY_PATH=${LD_LIBRARY_PATH} "$@"
```

Comment:
What is this for?

Reply:
In `.cargo/config.toml` you can specify a test runner.
Normally you could specify `sudo -E`, and all the tests would be run with `sudo`.

In our case, it's a bit more complicated because of wasmedge.

When we run the wasmedge tests without static linking, the binary uses `LD_LIBRARY_PATH` to find the shared library.
However `sudo` likes to block `LD_LIBRARY_PATH` even whith `-E` argument, so we need to set it manually.

So this is just a script that we can specify as the test runner that does `sudo` and forwards `LD_LIBRARY_PATH`.

If you look [here](https://github.com/containerd/runwasi/blob/main/crates/containerd-shim-wasm/src/sandbox/testutil.rs#L37-L48) we are already doing this in `run_test_with_sudo`.